scripts/reporting/create_doi.py

```diff
@@ -0,0 +1,100 @@
+'''
+Programmatically create DOI for a Synapse project
+'''
+import json
+import os
+import requests
+import yaml
+
+''' Constants '''
+OBJECT_ID = 'syn25191261'
```

Comment:
This should be fed in via a CLI argument, not a constant - see `argparse` use elsewhere